WorkflowUI/Sources/Container/ContainerViewController.swift

```diff
@@ -69,8 +75,10 @@
             fatalError("init(coder:) has not been implemented")
         }
 
-        private func render(screen: ScreenType, environment: ViewEnvironment) {
-            rootViewController.update(screen: screen, environment: environment)
+        private func update(screen: ScreenType, environment: ViewEnvironment) {
```

Comment:
Do we need to also call this from `viewDidLoad` to ensure the views get added etc.?

Reply:
We do not – when the container loads its view, it will add the rootVC as a child: https://github.com/square/workflow-swift/blob/main/WorkflowUI/Sources/Container/ContainerViewController.swift#L81